policy/policy.go

```diff
@@ -293,7 +332,32 @@ type Resource struct {
 // level accept. match indicate a rule level match
 // and credential acceptance.
 func (r *Resource) Match(r1 *Resource) (bool, bool) {
-	accept, match := r.Name.Match(r1.Name)
+	// When matching an input resource against a resource defined by the policy, just consider
```

Comment:
is it possible fur us to put this blanking of the resource as the policy level?  I think it would be useful to be able to do strict matching by default, but then apply in the context of a policy more loosely.  That would also allow us to build and store the field map inside the policy object (obviously it would be a private, unserialized entity) and only have to do it once per policy when we load that policy.

Reply:
The reason it is at the resource level is that the policy has the `allowed` and `disallowed` resource sets. Having it at the policy level would mean having the field list for each resource in both sets.
We can still store the private field map in the resource struct and process it just once, when creating the policy.